Dockerfile

```diff
@@ -4,21 +4,17 @@ ARG DOCKER_ARCH=amd64
 ### Deps stage ###
 ##################
 
-FROM $DOCKER_ARCH/rust:slim-buster AS relay-deps
+FROM $DOCKER_ARCH/rust:alpine AS relay-deps
 
 ARG DOCKER_ARCH
 ARG BUILD_ARCH=x86_64
 
 ENV DOCKER_ARCH=${DOCKER_ARCH}
 ENV BUILD_ARCH=${BUILD_ARCH}
 
-ENV BUILD_TARGET=${BUILD_ARCH}-unknown-linux-gnu
+ENV BUILD_TARGET=${BUILD_ARCH}-unknown-linux-musl
 
-RUN apt-get update \
-    && apt-get install --no-install-recommends -y \
-    curl build-essential git zip cmake \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+RUN apk add --no-cache curl gcc g++ git cmake make bash musl-dev perl zip
```

Comment:
For the bits built with g++, are you sure those link against musl? [In sentry-cli](https://github.com/getsentry/sentry-cli/blob/master/Dockerfile#L3) we use `getsentry/rust-musl-cross` as base container, which also sets the `CC` and `CXX` variables to cross compilers.

Reply:
I looked up `rust-musl-cross` (didn't know we had it) and it appears to use Ubuntu as base image, which necessitates using cross compilers. I used Alpine, which doesn't carry glibc anywhere at all, only musl.